src/components/language/language.service.ts

```diff
@@ -55,6 +58,18 @@ import { EthnologueLanguageService } from './ethnologue-language';
 
 @Injectable()
 export class LanguageService {
+  private readonly securedProperties: string[] = [
+    'name',
+    'displayName',
+    'isDialect',
+    'populationOverride',
+    'registryOfDialectsCode',
+    'leastOfThese',
+    'leastOfTheseReason',
+    'displayNamePronunciation',
+    'sponsorDate',
+  ];
```

Comment:
Sorry for the confusion. I meant move the object here https://github.com/SeedCompany/cord-api-v3/blob/e7f1230fd74a03ddf60f8885371c2ebf9cf36866/src/components/language/language.service.ts#L425-L433
to class prop. Then
```tsx
sort in this.securedProperties
```